src/main/Sirius.java

```diff
@@ -0,0 +1,291 @@
+package main;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.lang.Integer;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import task.*;
+import exception.*;
+
+public class Sirius {
+    // some commands
+    public static final String BYE = "bye";
+    public static final String LIST = "list";
+    public static final String MARK = "mark";
+    public static final String UNMARK = "unmark";
+    public static final String TODO = "todo";
+    public static final String DEADLINE = "deadline";
+    public static final String EVENT = "event";
+    public static final String DELETE = "delete";
+
+
+    // some regexes
+    public static final String SPACE = " ";
+    public static final String EMPTY = "";
+    public static final String SLASH = "/";
+    public static final String STATUS_DELIMINATOR = "\\|";
+    public static final String SEPARATOR = "-----------------------------";
+
+    // data members
+    private static boolean isExit = true;
+    private static boolean isValidToProcess = true;
+    private static final ArrayList<Task> list = new ArrayList<>();
+
+    // some methods
+    public static void sayHello(){
+        System.out.println("""
+                -----------------------
+                Hello! I'm Sirius!
+                What can I do for you?
+                -----------------------
+                """);
+    }
+    public static void sayGoodbye(){
+        System.out.println("""
+                -----------------------
+                Bye! Hope to see you soon.
+                -----------------------
+                """);
+        isExit = false;
+    }
+    public static void markTask(String[] commandPieces, ArrayList<Task> list, boolean isMarked) {
+        System.out.println(SEPARATOR);
+        try {
+            int taskNumber = Integer.parseInt(commandPieces[1]);
+            if (isMarked) {
+                list.get(taskNumber - 1).setMarked(true);
+                System.out.println("Nice! I've marked this task as done:");
+            } else {
+                list.get(taskNumber - 1).setMarked(false);
+                System.out.println("OK, I've marked this task as not done yet:");
+            }
```

Comment:
Consider setting the 'isMarked' variable outside the conditional to reduce duplicate code